src/common/ActionButtons/ActionButtons.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+	IWebchatButton,
+	IWebchatQuickReply,
+} from "@cognigy/socket-client/lib/interfaces/messageData";
+import { ActionButton } from ".";
+import classnames from "classnames";
+
+import classes from "./ActionButtons.module.css";
+import { FC } from "react";
+import { MessageProps } from "src/Message";
+
+export interface ActionButtonsProps {
+	action?: MessageProps["action"];
+	payload: IWebchatButton[] | IWebchatQuickReply[];
+	containerClassName?: string;
+	buttonClassName?: string;
+}
+
+const ActionButtons: FC<ActionButtonsProps> = props => {
+	const buttons = props.payload.filter((button: ActionButtonsProps["payload"][number]) => {
+		if ((button as any).type === "element_share") return false;
```

Comment:
don't we have the correct type here instead of "any"?

Reply:
We have, though they would need to be fixed first. I left this out in this PR as it's out of scope.